### ISSUE TITLE
"Warning: 'tube_extremes' is deprecated, using 'capping' instead..."

### ISSUE THREAD

**Issue by rtoy** [rtoy/maxima#3655]
Imported from SourceForge on 2024-07-08 11:05:35
Created by **[peterpall](https://sourceforge.net/u/peterpall/)** on 2024-05-12 04:12:46
Original: https://sourceforge.net/p/maxima/bugs/4304

---

When trying to run all the examples from the draw() manual through gnuplot 6 I encountered one warning. Surprisingly that warning was created by maxima, not by gnuplot:
~~~
"Warning: 'tube_extremes' is deprecated, using 'capping' instead..."
~~~

**Comment by rtoy** [comment]
Imported from SourceForge on 2024-07-08 11:05:37
Created by **[peterpall](https://sourceforge.net/u/peterpall/)** on 2024-05-12 04:20:01
Original: https://sourceforge.net/p/maxima/bugs/4304/#0e48

---

Resolved with commit 727cd8902

**Comment by rtoy** [comment]
Imported from SourceForge on 2024-07-08 11:05:40
Created by **[peterpall](https://sourceforge.net/u/peterpall/)** on 2024-05-12 04:20:17
Original: https://sourceforge.net/p/maxima/bugs/4304/#0930

---

- **status**: open --> closed
